### ISSUE TITLE
remove solar charger updating. Fixes venus v2.0x issues

### ISSUE THREAD

**Issue by mpvader** [izak/bms-bridge#4]
bms-bridge doesn't properly work against Venus v2.00. The reason is that dbus-systemcalc is now also writing to /link/network mode. Which is a bit mask, so dbus-systemcalc & bms-bridge were undoing each-others changes. The result is that solarchargers show `Error #67` again.

A fix is coming: Venus v2.01 has a new version of dbus-systemcalc that also takes care of writing the bms/max-charge-current to the solar-chargers.

https://github.com/victronenergy/dbus-systemcalc-py/commit/af23489c77435a8225910b257461978d28e3c2c8

So what needs to be done is removing the lines of code related to writing the max-charge-current. Which leaves only the code that sets a Multi to Charger-only when the max-discharge-current == 0.

**Comment by mpvader** [comment]
Note that the conclusion of all of this is that Venus v2.00 is not compatibel to any version of bms-bridge.

And only Venus v2.01, once released, will be compatible again with a version of bms-bridge which is still to be finalized

**Comment by mpvader** [comment]
This is still open, after adding BMS-CCL control over VE.Direct Solar Chargers in Venus v2.01, we removed it again in Venus v2.03 since it was giving problems. The details:

1) bms-bridge used to do CCL=0 control, for VE.Direct chargers only, but Venus v2.00 broke that. See above.
2) then we implemented it in Venus itself, v2.01, containing systemcalc v1.36. See [1] for details. Basically it copied the CCL limit received from the battery to each VE.Direct MPPT solar charger. So, when CCL=0, all MPPTs would be disabled entirely. And if CCL=1, all MPPTs would be set to 1A max output, etc.
3) at the same time, the CCL=0 function was removed from bms-bridge. [2].
4) then we found that limiting the charge current of MPPTs causes problems. And removed that feature again, see [3]. This was part of systemcalc v1.39, part of Venus v2.03.

And that is were we are right now: the CCL=0 function is no longer in bms-bridge, its also not easy at all to  put it back since it conflicts with systemcalc itself. And the CCL=0 function is also no longer in Venus.

So, long story short, to have CCL=0 right now, use v2.02 in combination with bms-bridge. Anything later will not have the CCL=0 function for VE.Direct Solar Chargers.


[1] https://github.com/victronenergy/dbus-systemcalc-py/commit/af23489c77435a8225910b257461978d28e3c2c8
[2] https://github.com/izak/bms-bridge/commit/7774e333906367b341acdf35530ca93598a8bf42
[3] https://github.com/victronenergy/dbus-systemcalc-py/commit/83359160cc43b5febc26f69666c7187b2e757250
